ractor_actors/src/net/tcp/mod.rs

```diff
@@ -90,36 +89,3 @@ pub enum IncomingEncryptionMode {
     /// Accept sockets and establish a secure connection
     Tls(tokio_rustls::TlsAcceptor),
 }
-
-/// Represents a receiver of frames of data
-#[cfg_attr(feature = "async-trait", async_trait::async_trait)]
-pub trait FrameReceiver: ractor::State {
-    /// Called when a frame is received by the [TcpSession] actor
-    #[cfg(not(feature = "async-trait"))]
-    fn frame_ready(
-        &self,
-        f: Frame,
-    ) -> impl std::future::Future<Output = Result<(), ActorProcessingErr>> + Send;
-
-    /// Called when a frame is received by the [TcpSession] actor
-    #[cfg(feature = "async-trait")]
-    async fn frame_ready(&self, f: Frame) -> Result<(), ActorProcessingErr>;
-}
-
-/// Represents
-#[cfg_attr(feature = "async-trait", async_trait::async_trait)]
-pub trait SessionAcceptor: ractor::State {
-    /// Called when a new incoming session is received by a `TcpListener` actor
-    #[cfg(not(feature = "async-trait"))]
-    fn new_session(
-        &self,
-        session: NetworkStream,
-    ) -> impl std::future::Future<Output = Result<(), ActorProcessingErr>> + Send;
-
-    /// Called when a new incoming session is received by a `TcpListener` actor
-    #[cfg(feature = "async-trait")]
-    async fn new_session(&self, session: NetworkStream) -> Result<(), ActorProcessingErr>;
-}
-
-pub use listener::*;
```

Comment:
yeah this specifically I think will be a semver break. Not the end of the world, but can we re-export the types at the module root for `tcp`?

Reply:
Sure. I guess it's expected that people only want to import the tcp module as today, so I guess `session::*` should be exported too?

Reply:
I've re-exported both listener and session symbols to the tcp module, `cargo semver-checks check-release` shows no breakage now.